sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -37,6 +37,10 @@ void _auto_configure_addr(gnrc_netif_t *netif, const ipv6_addr_t *pfx,
     DEBUG("nib: add address based on %s/%u automatically to interface %u\n",
           ipv6_addr_to_str(addr_str, pfx, sizeof(addr_str)),
           pfx_len, netif->pid);
+    if (!(netif->flags & GNRC_NETIF_FLAGS_HAS_L2ADDR)) {
```

Comment:
I don't think was initially meant to be part of this PR ;)

Reply:
Oops, yes!

Reply:
(I moved that fix to #10483 ;-))